otherlibs/stdune/src/loc0.ml

```diff
@@ -1,19 +1,133 @@
-include Lexbuf.Loc
+type t =
+  | None_loc
+  | In_file of string
+  | Loc of Lexbuf.Loc.t
```

Comment:
And this one could be `Lexbuf_loc`/`Lexbuf_location`?

Reply:
Lexbuf_loc it is.